### DISCUSSION TITLE
Upgrade samples to Dot Net 6

### DISCUSSION THREAD

**Discussion by wpschaub** [WorkSafeBC-Common-Engineering/Common.Engineering.Discussion#3] [Updates - Templates]
@said-akram any chance you can submit a pull request for these issues, while I work with @604kev to get the toolbox and templates published?

- https://github.com/WorkSafeBC-Common-Engineering/AzureDevOps.Automation.Pipeline.Sample.azure-function/issues/3
- https://github.com/WorkSafeBC-Common-Engineering/AzureDevOps.Automation.Pipeline.Sample.generic-multiple-jobs/issues/2
- https://github.com/WorkSafeBC-Common-Engineering/AzureDevOps.Automation.Pipeline.Sample.generic-single-job/issues/3

We can implement them internally and regression test through our self-service automation. cc @danielrbroderick

**Comment by said-akram** [comment]
Hi @wpschaub 
I just submitted a [PR](https://github.com/WorkSafeBC-Common-Engineering/AzureDevOps.Automation.Pipeline.Sample.azure-function/pull/4) for sample azure-function.
I will try to work on the other two tomorrow.

**Reply by wpschaub** [reply]
`THANK YOU` @said-akram you are a STAR!

@danielrbroderick @MerlinDarkstar please action the new pull requests. I have tested the updates with our in-house v2 blueprints.
